card/db/schema.rb

```diff
@@ -66,30 +68,48 @@
   add_index "card_revisions", ["creator_id"], name: "revisions_created_by_index", using: :btree
 
   create_table "cards", force: :cascade do |t|
-    t.string   "name",                limit: 255,      null: false
-    t.string   "key",                 limit: 255,      null: false
-    t.string   "codename",            limit: 255
     t.integer  "left_id",             limit: 4
-    t.integer  "right_id",            limit: 4
-    t.integer  "current_revision_id", limit: 4
     t.datetime "created_at",                           null: false
     t.datetime "updated_at",                           null: false
+    t.integer  "current_revision_id", limit: 4
+    t.string   "name",                limit: 255,      null: false
     t.integer  "creator_id",          limit: 4,        null: false
     t.integer  "updater_id",          limit: 4,        null: false
+    t.integer  "right_id",            limit: 4
+    t.string   "key",                 limit: 255,      null: false
+    t.boolean  "trash",                                null: false
+    t.integer  "references_expired",  limit: 4
+    t.string   "codename",            limit: 255
     t.string   "read_rule_class",     limit: 255
     t.integer  "read_rule_id",        limit: 4
-    t.integer  "references_expired",  limit: 4
-    t.boolean  "trash",                                null: false
     t.integer  "type_id",             limit: 4,        null: false
     t.text     "db_content",          limit: 16777215
   end
 
+  add_index "cards", ["created_at"], name: "cards_created_at_index", using: :btree
   add_index "cards", ["key"], name: "cards_key_index", unique: true, using: :btree
   add_index "cards", ["left_id"], name: "cards_left_id_index", using: :btree
   add_index "cards", ["name"], name: "cards_name_index", using: :btree
   add_index "cards", ["read_rule_id"], name: "cards_read_rule_id_index", using: :btree
   add_index "cards", ["right_id"], name: "cards_right_id_index", using: :btree
   add_index "cards", ["type_id"], name: "cards_type_id_index", using: :btree
+  add_index "cards", ["updated_at"], name: "cards_updated_at_index", using: :btree
+
+  create_table "delayed_jobs", force: :cascade do |t|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.